include/wallaby/tello.h

```diff
@@ -0,0 +1,23 @@
+
+#ifndef INCLUDE_WALLABY_TELLO_H
+#define INCLUDE_WALLABY_TELLO_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+char * find_tellos();
+
+int connect_tello(char * tello);
+
+int wpa_cmd(char const  * cmd, char * buf);
+
+int send_to_tello(char * command);
```

Comment:
Shouldn't this be const?

Reply:
Also, consider making this `tello_send` to be consistent with rest of libwallaby.